File: Forms/frmQR.vb
Public Class frmQR
    Private Sub frmQR_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Try
            PictureBox1.Image = Image.FromFile(strFilename)
        Catch ex As Exception
        End Try
    End Sub

    Private Sub Menu_Exit_Click(sender As Object, e As EventArgs) Handles Menu_Exit.Click
        PictureBox1.Image = Nothing
        Me.Dispose()
    End Sub

    Private Sub PictureBox1_DoubleClick(sender As Object, e As EventArgs) Handles PictureBox1.DoubleClick
        Menu_Exit_Click(sender, e)
    End Sub
End Class

File: Forms/frmQR.cs
using System;
using System.Drawing;

namespace eLib
    {
    public partial class frmQR
        {
        public frmQR ()
            {
            InitializeComponent ();
            }
        private void frmQR_Load (object sender, EventArgs e)
            {
            try
                {
                PictureBox1.Image = Image.FromFile (eLibFile.Filename);
                }
            catch (Exception ex)
                {
                }
            }
        private void Menu_Exit_Click (object sender, EventArgs e)
            {
            PictureBox1.Image = null;
            Dispose ();
            }
        private void frmQR_KeyDown (object sender, System.Windows.Forms.KeyEventArgs e)
            {
            if (((int) e.KeyCode == 27) | ((int) e.KeyCode == 13))
                {
                Menu_Exit_Click (null, null);
                }
            }
        private void PictureBox1_DoubleClick (object sender, EventArgs e)
            {
            Menu_Exit_Click (sender, e);
            }
        private void lblExit_Click (object sender, EventArgs e)
            {
            Menu_Exit_Click (sender, e);
            }
        }
    }